src/composables/multisigAccounts.ts

```diff
@@ -202,19 +202,19 @@ export function useMultisigAccounts({ store, pollOnce = false }: MultisigAccount
               signers,
               consensusResult,
               balance,
-            ] = await Promise.all([
+            ] = (await Promise.all([
               (
                 (isAdditionalInfoNeeded.value && gaAccountId === activeMultisigAccountId.value)
                 || currentAccount?.nonce == null
               )
-                ? contractInstance.methods.get_nonce()
+                ? contractInstance.get_nonce()
                 : { decodedResult: currentAccount.nonce },
               currentAccount?.signers
                 ? { decodedResult: currentAccount.signers }
-                : contractInstance.methods.get_signers(),
-              contractInstance.methods.get_consensus_info(),
-              gaAccountId ? sdk.balance(gaAccountId) : 0,
-            ]);
+                : contractInstance.get_signers(),
+              contractInstance.get_consensus_info(),
+              gaAccountId ? sdk.getBalance(gaAccountId) : new Promise(() => 0),
+            ])) as any;
```

Comment:
I think this is bad to treat all resolved promises as `any`. Maybe you can add some typings?